src/containers/Folder.jsx

```diff
@@ -24,14 +37,37 @@ class Folder extends Component {
 
   render (props, state) {
     if (props.isFetching === true) {
-      return <p>Loading</p>
+      return (
+        <div role='contentinfo'>
+          <p>Loading</p>
+        </div>
+      )
     }
-    return <FileList {...props} {...state} />
+    const isTrashContext = props.context === TRASH_CONTEXT
```

Comment:
Regarding your other comments, this refactoring is a WIP, there's still a lot to do in my mind to make things nicer. But let's merge this one so that we can move the card ;)